src/main/java/com/gizasystems/automationexercise/pages/Footer.java

```diff
@@ -35,6 +28,11 @@ public Footer clickOnSubscribeButton() {
         return this;
     }
 
+    @Step("Scroll to bottom page")
+    public Footer scrollToBottomPage() {
+        driver.element().scrollToElement(subscriptionText);
+        return this;
+    }
     //////////////////// Validations \\\\\\\\\\\\\\\\\\\\
```

Comment:
Please add an empty line here between the method and the Validation section.

Reply:
Done